### PR TITLE
N clusters creation

### PR DESCRIPTION
**Pull request by iranzo**

### CODE REVIEW

File: hack/deploy-hub-local/create-vm.yml [hunk 1]
```diff
@@ -10,10 +10,10 @@ spoke{{ i }}-m{{ j }}:
   numcpus: 8
   nets:
   - name: bare-net
-    mac: ee:ee:ee:ee:ee:{{ j }}e
+    mac: ee:ee:ee:ee:{{ i }}:{{ j }}e
   - name: kubeframe
     nic: enp2s0
-    mac: aa:aa:aa:aa:aa:{{ j }}a
+    mac: aa:aa:aa:aa:{{ i }}:{{ j }}a
```
Comment (alknopfler) [inline hack/deploy-hub-local/create-vm.yml:16]: i should be 2 so add like j

File: hack/deploy-hub-local/build.sh [hunk 2]
```diff
@@ -53,19 +52,58 @@ if [ "$OC_DEPLOY_METAL" = "yes" ]; then
...
+
+#Empty file before we start
+
+>spokes.yaml
+
+CHANGE_IP=$(kcli info vm test-ci-installer | grep ip | awk '{print $2}')
+
+for spoke in $(seq 1 $CLUSTERS); do
+
+    SPOKE1_0=$(kcli info vm spoke${spoke}-m0 | grep id | awk '{print $2}')
```
Comment (alknopfler) [inline hack/deploy-hub-local/build.sh:69]: Should be spoke_0 right?

**Review by alknopfler — CHANGES_REQUESTED** [review]

**Comment by alknopfler** [comment]
Revised @iranzo with this change should lgtm

File: hack/deploy-hub-local/build.sh (hunk 2 above)
Comment (iranzo) [reply hack/deploy-hub-local/build.sh]: it's generic, replaced to MASTER_0

**Review by alknopfler — APPROVED** [review]

**Pull request merged**